src/main/kotlin/io/libp2p/pubsub/gossip/GossipParams.kt

```diff
@@ -10,62 +10,222 @@ import kotlin.math.min
 
 typealias Weight = Double
 
-data class GossipParamsCore(
+/**
+ * Parameters of Gossip 1.1 router
+ */
+data class GossipParams(
+    /**
+     * [D] sets the optimal degree for a GossipSub topic mesh. For example, if [D == 6],
+     * each peer will want to have about six peers in their mesh for each topic they're subscribed to.
+     * [D] should be set somewhere between [DLow] and [DHi].
+     */
     val D: Int = 3,
```

Comment:
nit: Isn't this a bit low?

Reply:
Right 👍 
`3` is some ancient default. `6` is in the Go impl